assets/pages/contributions-landing/contributionsLanding.scss

```diff
@@ -13,8 +13,8 @@
 @import '../../components/contribute/contribute';
 @import '../../components/pageSection/pageSection';
 @import '../../components/secure/secure';
-@import '../../containerisableComponents/contributionAwarePaymentLogos/contributionAwarePaymentLogos';
-@import '../../containerisableComponents/contributionSelection/contributionSelection';
+@import '../../components/contributionAwarePaymentLogos/contributionAwarePaymentLogos';
```

Comment:
Since you are here... it is possible to replace `'../../components/...` with `~components/...`